app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+	
+<hi>About</hi>
+<p>
+	The <a href="http://www.railstutorial.org"><em>Ruby on Rails Tutorial</em></a>is a
```

Comment:
Khoảng cách đầu dòng so với dòng trên là 2 spaces thôi tương tự cho toàn bộ file này